doc/source/ray-core/gpu-objects.rst

```diff
@@ -0,0 +1,212 @@
+.. _gpu-objects:
+
+GPU Objects
+===========
+
+Ray objects are normally stored in Ray's CPU-based object store and are copied to GPU memory as needed by the user.
+This can lead to unnecessary and expensive data transfers.
+For example, passing a CUDA ``torch.Tensor`` from one Ray task to another would require a copy from GPU to CPU memory, then back again to GPU memory.
+
+GPU objects are a new feature that allows Ray to store and pass objects directly in GPU memory, enabling zero-copy or near-zero-copy data transfers.
```

Comment:
"zero-copy data transfers" seems a little misleading/confusing -- there's still one copy from GPU<>GPU

maybe be a bit more direct and say "using optimized transports that avoid unnecessary copies to CPU memory"

Reply:
actually, I'd suggest just deleting this sentence. it's redundant with the below, which is more precise

Reply:
It is zero-copy if it gets passed back to the same actor, but will reword.